### ISSUE TITLE
[Cityscape] [Scoring] Damage to the city caused by other organisations can be held against X-COM

### ISSUE THREAD

**Issue by FilmBoy84** [OpenApoc/OpenApoc#776]
Apart from damage from aliens (which should count)

Damage from all other organisations should not count against X-COM but that organisations own score (hidden from player) and affect only their funding

To replicate, start as x-com, put time on SLOW

Cheat forward several weeks, and allow the UFOs to spawn in
Sooner or later they'll do a bombing run, which will annoy an organisation like marsec or megapol

Observe that when a none-alien craft hits a building, that adds to the negative score for damage to the city

Alien craft causing damage SHOULD count however

![image](https://user-images.githubusercontent.com/33913088/65234060-ce8bde80-dacb-11e9-882e-de5e9afa8f10.png)

**Comment by Quickmind01** [comment]
It appears the game can tell who hits a building. Is that when the score is affected? If so, that could be where a check is made before adjusting the score. Would organizations under alien control still not affect X-COM?

**Comment by FilmBoy84** [comment]
Yes, score is affected at the moment of each strike and instance of damage in the original game IIRC

As to organisations under alien control, i do believe this DOES affect X-COM score as that organisation counts as an extension of the alien influence. But we would have to check this in OG to confirm (It's not easy to setup without debug tools)
